### ISSUE TITLE
Layers' settings (polygons, transparency) not saving

### ISSUE THREAD

**Issue by v4sk1n** [osmandapp/OsmAnd#14703]
### Description
When I switch underlays or overlays, or just restart osmand - layers' settings (polygons, transparency) are not saving
Each time, I have to manually enter "map settings", and enable "polygons" setting of underlay manually
In previous osmand version, it saved the "polygons" setting automatically

### How to reproduce?
Switch underlays or overlays, or just restart osmand

### Your Environment
OsmAnd Version: 4.2.5
Android/iOS version: Android 10
Device model: Honor 9A

**Maps used (online or offline):**   
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.
Online (underlays, overlays)

**Comment by Lans-K** [comment]
Yes, and the same thing will happen when changing the screen orientation to landscape mode and back.

**Comment by Zirochkabila** [comment]
> Yes

Exactly right, when you change the orientation of the screen the bug repeats itself

**Comment by Lans-K** [comment]
In the release version 4.2.6, my problem has not disappeared.

**Comment by vshcherb** [comment]
Only in 4.2.7
